src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/AzureRM.DataLakeAnalytics.psd1

```diff
@@ -0,0 +1,90 @@
+#  
+# Module manifest for module 'Microsoft.Azure.Commands.DataLakeAnalytics'  
```

Comment:
Why this project have two psd1 files? For PowerShellGet purposes we just need the one under DataLakeAnalytics folder.
https://github.com/begoldsm/azure-powershell/blob/dev/src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/AzureRM.DataLakeAnalytics.psd1
https://github.com/begoldsm/azure-powershell/blob/dev/src/ResourceManager/DataLakeAnalytics/AzureRM.DataLakeAnalytics.psd1